www/src/pages/insights.tsx

```diff
@@ -17,6 +17,7 @@ import SummaryUsageExamples, { ISummaryUsageExamples } from '../components/insig
 import ProjectUsageExamples, { IProjectUsageExamples } from '../components/insights/ProjectUsageExamples';
 import ComponentUsageExamples, { IComponentUsageExamples } from '../components/insights/ComponentUsageExamples';
 import getGithubProjectUrl from '../utils/getGithubProjectUrl';
+// @ts-ignore
 import dependentProjectsAnalysis from '../../../dependent-usage.json'; // eslint-disable-line
```

Comment:
typescript complains that we export a file outside of `rootDIr` specified in `tsconfig.json`, I didn't find an easy fix for this except for moving the file to `www` project, but I'm not sure that we should do that

Reply:
Hmm, yeah I don't think we'll be able to do that, at least not trivially. Good call to ignore for now.